### DISCUSSION TITLE
White List doesn't work

### DISCUSSION THREAD

**Discussion by namoi76** [nefarius/HidHide#41] [Q&A]
For whatever reason, white listing doesn't work on my machine.  At first Install it seems t work, but as soon as I reboot after the first time I install HidHide, the white listed apps doesn't see the devices anymore.
I am on W10 Pro. Not sure what info I could give to help nail the problem though.
The app I tried to white list are Joystick Gremlin and VKB config software.

As soon as I uncheck the hidden devices, they reappear in these apps.

Thanks for any insights.

**Comment by Chanticrow** [comment]
I was assisting a friend with the exact same issue.  We found that a program, Zone Alarm, was preventing HIDHide from implementing the whitelist.  Check your antivirus.
